Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/prometheus/golang-builder AS builder
+
+# Get sql_exporter
+ADD .   /go/src/github.com/free/sql_exporter
+WORKDIR /go/src/github.com/free/sql_exporter
+
+# Do makefile
+RUN make
+
+# Make image and copy build sql_exporter
+FROM        quay.io/prometheus/busybox:glibc
+MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
```

Comment:
I am not actually an (official) Prometheus developer, so I'm not sure this is completely right, but I also don't know if it really matters.